index.html

```diff
@@ -0,0 +1,213 @@
+                </div>
+            </div>
+        </nav>
+        <nav class="navbar navbar-expand-lg bg-body-tertiary">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#"><img src="./images/aseel-logo.webp" alt="logo" id="logo"></a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span id="closeNav" class="fa fa-bars"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNav">
+                    <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="active" aria-current="page" href="about.html">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Join</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link aseel-name p-1">Aseel Afghanistan</a>
+                    </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <!-- FIRST BODY SECTION START HERE -->
+    <section class="first-section d-flex align-items-center">
+        <div class="container">
+            <div class="first-section-content">
+                <div class="contant">
+                    <h2 class="first-color">"Hello! Sharing World"</h2>
+                </div>
+                <div class="h1-title">
+                    <h1>Aseel | Buy Good, Do Good</h1>
+                </div>
+                <div class="first-p">
+                    <p>Aseel is a tech-based startup company providing a digital platform that allows artisans to sell their antique handcrafted products and provide humanitarian aid to and from anywhere in the world with complete transparency. We believe in the practical use of the skills to build the positive change the businesses and communities need. The easy and user-friendly platform of Aseel supports thousands of makers by connecting them to global buyers. Transparency and privacy have always been the core values of our team. We are committed to supporting our clients through our secured and protected customer service page anytime.</p>
+                </div>
+                <div class="pt-5 fourth-color">
+                    <h4 class="h4-title">2023.10.15(THU) ~ 16(FRI)</h4>
+                </div>
+                <div class="pt-1 fourth-color">
+                    <h6>@ National Museum of Kabul-Afghanistan, Shahr-E-Naw</h6>
+                </div>
+            </div>
```

Comment:
- [ ] It will be better to give some margin top in mobile version because the header in this section is hidden due to navbar. You can check your design below
>![image](https://user-images.githubusercontent.com/53402600/234520508-10159c0c-df56-40d0-970b-0c749f407ab8.png)